src/page-multi-facility/MultiFacilityImpactDashboard.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect, useState } from "react";
```

Comment:
I don't believe this file should be included in this commit since all of this code can already be found in the `MultiFacilityPage`.  Perhaps it's from another branch you were working on locally?

Reply:
Sorry, I see what's going on now.  You toggle the dashboard page and the scenario creation page.  Ignore this comment

Reply:
Oh I actually added this so I could separate this component from the `<MultiFacilityPage>` component and display either the Multi-Facility page OR the `CreateBaselineSceanarioPage` depending on whether it has a baseline scenario.

Reply:
Yup, I commented before reading the whole review 🙈.  Sorry!